paparazzi/src/main/java/app/cash/paparazzi/DeviceConfig.kt

```diff
@@ -76,6 +76,7 @@ data class DeviceConfig(
   val keyboardState: KeyboardState = KeyboardState.SOFT,
   val softButtons: Boolean = true,
   val navigation: Navigation = Navigation.NONAV,
+  val screenRound: com.android.resources.ScreenRound? = null,
```

Comment:
should this default to NOTROUND, or is that different from null?

Reply:
Not sure, I wanted to retain the current behaviour by default.  It's currently null to HardwareConfig, not NOTROUND.

Reply:
I see; that being said, it'd be nice to avoid nullable types and instead set rely on reasonable defaults.  Let's set to NOTROUND.